corehq/apps/app_manager/models.py

```diff
@@ -2199,6 +2199,11 @@ def wrap(cls, data):
         return self
 
 
+class ShadowFormMapping(DocumentSchema):
+    form_id = StringProperty()
+    session_endpoint_id = StringProperty()  # See toggles.SESSION_ENDPOINTS
```

Comment:
Is the intention to expand the use of this class beyond session endpoints?  I saw something on the spec about moving `excluded_form_ids` here.

Reply:
I thought I could do that. But after thinking it through I realized that if I switch from and exclusive list to an inclusive list that will change the behavior when adding new forms to the source module from being added by default to being left out by default. After talking to Jenny that is not something we want. Since a class ShadowForm already exists what do you think about ShadowModuleForm to match you nit for `form_session_endpoints` above?

Reply:
Maybe something with `endpoint` in the name?  Like, `ShadowFormEndpoint`?